apps/charterafrica/src/pages/index.page.js

```diff
@@ -42,6 +42,55 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
     fallbackLocale: defaultLocale,
   });
 
+  const { docs: homePage } = await payload.findPage("home", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  const { blocks: pageBlocks } = homePage[0] ?? {
+    blocks: [],
+  };
+
+  const blocks = pageBlocks.map(({ blockType, ...other }) => ({
+    ...other,
+    slug: blockType,
+  }));
+
+  const spotlightBlock = blocks.find((block) => block.slug === "spotlight");
+
+  const spotlight = {
+    title: "Spotlight",
+  };
+
+  Object.keys(spotlightBlock).forEach((key) => {
+    if (key === "items") {
+      spotlight[key] = spotlightBlock[key].map((spotlightItem) => {
```

Comment:
Why not just
```js
  const items = spotlightBlock.items?.map((s) => ...
    ...
  spotlight.items = items || null;
```

`Object.keys`, then `.forEach`, then `if`, and then `map` is very hard to read and understand.